ang/civicase/case/search/directives/search.directive.js

```diff
@@ -61,6 +61,14 @@
         'id': 'is_involved'
       }
     ];
+    var contactRoleFilters = [
```

Comment:
Why do we need to save it in a separate variable and then assign to `$scope`? cant we do it directly?

Reply:
Because we need to have the default value selected which is `['all-case-roles']`.

The alternative would be this:

```js
  $scope.contactRoleFilters = [
      { id: 'all-case-roles', text: 'All Case Roles' },
      { id: 'client', text: 'Client' }
    ];
  $scope.searchForm = {
    selectedContacts: [],
    selectedContactRoles: [ 'all-case-roles' ]
  };
```

Does not look too bad, but I need to check if this is not going to affect the URL binding.

Reply:
This looks better, please check if this works.